sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectKey=twilio_twilio-cli
```

Comment:
[Q] Where are we using these properties?

Reply:
This key is generated by Sonar dashboard, which needs to be a 1:1 mapping between repositories. 
Helps sonar to link a CI build to it's dashboard view.